src/Markup/Avalonia.Markup.Xaml/Templates/ItemsPanelTemplate.cs

```diff
@@ -4,14 +4,14 @@
 
 namespace Avalonia.Markup.Xaml.Templates
 {
-    public class ItemsPanelTemplate : ITemplate<Panel>
+    public class ItemsPanelTemplate : ITemplate<Panel?>
```

Comment:
Is this correct? IIRC an items panel template shouldn't return null and the `ItemsControl.ItemsPanel` property expects a non-null `Panel`.

Reply:
Good one. I annotated it with `null` because `TemplateContent.Load` in `Build()` can definitely return `null`.
- Should it throw on null content instead? That would be inconsistent with other templates, but quite logical.
- Alternatively, `ItemsControl.ItemsPanel` can be a `ITemplate<Panel?>` and the only real usage in `ItemsPresenter` checked for null. This also matches the WPF behavior.

Reply:
Yeah I guess go with WPF's behavior in this case then 👍 .